modules/mod_search/tmpl/default.php

```diff
@@ -21,6 +21,8 @@
 
 			if (($browser->isBrowser('msie') && ($browserVersion < 10))
 				|| ($browser->isBrowser('mozilla') && ($browserVersion < 4))
+				|| ($browser->isBrowser('chrome') && ($browserVersion < 31))
+				|| ($browser->isBrowser('safari') && ($browserVersion < 7.1))
```

Comment:
According to http://caniuse.com/#feat=input-placeholder safari supports the placeholder attribute since version 5